### ISSUE TITLE
elFinder is very slow when get focus (enable)

### ISSUE THREAD

**Issue by phongtattuan** [Studio-42/elFinder#1055]
I created elFinder on a modal.
When the modal open, I click on a file, the browser is blocked for 3-5 seconds. 
After that, the file get selected.

After investigate the code, I found that the issue is because of this line on [elFinder.js#1525](https://github.com/Studio-42/elFinder/blob/2.x/js/elFinder.js#L1525)
```javascript
$('texarea:focus,input:focus,button').blur();
```
If a page have many input, this operation may take many times to process => browser blocked.

I think this operation is useless, because the controls will blur automatically when we click outside it.
We should remove this line. 
How do you think ?

Thanks.

**Comment by nao-pon** [comment]
@phongtattuan Thanks for your report.

How about replace to `document.activeElement && document.activeElement.blur();`

```javascript
$('texarea:focus,input:focus,button').blur();
```
to
```javascript
document.activeElement && document.activeElement.blur();
```

**Comment by phongtattuan** [comment]
Hi @nao-pon 

What is the purpose of this operation.
Because I think the element will be blur when we click on elFinder.
So this operation is no need.

**Comment by nao-pon** [comment]
If comment out `$('texarea:focus,input:focus,button').blur();` INPUT is not fire blur on click in elFinder CWD or CWD item. (Check on Firefox 39)

![Test with comment out](http://xoops.hypweb.net/uploads/elfinder/locked/elFinder_issue_1055.gif)

**Comment by phongtattuan** [comment]
Thanks for your explain. it seem `document.activeElement && document.activeElement.blur();` is the best solution. 
Hope this will be merged in next version.

**Comment by nao-pon** [comment]
Merged to 2.1 branch. Thanks. :+1:
